components/MessageBox.tsx

```diff
@@ -113,7 +113,7 @@ const MessageBox = ({ type = 'white', withIcon = false, isLoading, children, ...
           </Box>
         )}
 
-        <Box>{children}</Box>
+        <Box width="100%">{children}</Box>
```

Comment:
Same effect but perhaps cleaner way to make a div fill out remaining space when parent has `display: flex`:
```suggestion
        <Box flex={1}>{children}</Box>
```